README.markdown

```diff
@@ -185,11 +101,6 @@ You should now be able to run the tests successfully by typing:
 
     make test
 
-### Further steps
-
-* [Getting the Google Drive integration working locally](docs/google-drive.md)
-* [Installing your local lms app in Canvas](docs/canvas.md)
```

Comment:
Are you sure you want to get rid of *all* of this documentation? Even if some of it is no longer mandatory to know when getting the app ready for the first time, it looks like some of the information in there could still be useful down the line.

Reply:
I've put most of this back, re-phrased as "How to set up manually" if you can't use devdata for some reason (e.g. doing a prod setup). Seems unlikely to be maintained unfortunately